app/models/foreman_virt_who_configure/output_generator.rb

```diff
@@ -289,7 +291,7 @@ def proxy_strings
       output = ''
       output << "\n#{proxy_type}=#{sanitize_proxy(proxy.full_url)}" if proxy.present?
       output << "\nno_proxy=#{sanitize_proxy(no_proxy)}" if no_proxy.present?
-      output << "\nno_proxy=*" if proxy.blank? && no_proxy.blank?
+      output << "\nno_proxy=*" if proxy.blank? && no_proxy.present?
```

Comment:
Just wanted to double check that this is sane:

```
...
[global]
debug=0
interval=172800
oneshot=False

[system_environment]
no_proxy=192.168.0.0.1,potato,snakes
no_proxy=*
...
```

Multiple entries for `no_proxy` are safe? And if no HTTP proxy is selected, the user's input for `no_proxy` should be overridden to `*` ?

Reply:
We should only have the 1 entry, good catch let me go see what is going on with that.